inheritance/src/main/java/inheritance/Review.java

```diff
@@ -0,0 +1,65 @@
+package inheritance;
+
+public class Review
+{
+    public int stars;
+    public String body;
+    public String author;
+    public  Restaurant restaurant;
+    public Shop shop;
+    public Theater theater;
+    public String movie;
```

Comment:
Can you think of a way to clean this up? Each Review is only about only place, but you're creating storage space for a Restaurant, Shop, Theater...